include/openssl/asm_base.h

```diff
@@ -44,6 +47,22 @@
 .popsection
 #endif
 
+#if defined(__CET__) && defined(OPENSSL_X86_64)
+// Clang and GCC define __CET__ and provide <cet.h> when they support Intel's
+// Indirect Branch Tracking.
+// https://lpc.events/event/7/contributions/729/attachments/496/903/CET-LPC-2020.pdf
+//
+// cet.h defines _CET_ENDBR which is used to mark function entry points for IBT.
+// and adds the assembly marker. The value of _CET_ENDBR is made dependant on if
+// '-fcf-protection' is passed to the compiler. _CET_ENDBR is only required when
+// the function is the target of an indirect jump, but BoringSSL chooses to mark
+// all assembly entry points because it is easier, and allows BoringSSL's ABI
+// tester to call the assembly entry points via an indirect jump.
```

Comment:
Are there changes to the ABI tests in another commit? Or this is referring to something else?

Reply:
This says it "allows [the] tester to call the assembly entry points via an indirect jump", but I don't see any subsequent commits from BoringSSL that relate to this.